docs/configs/docsdev.js

```diff
@@ -161,6 +161,10 @@ export default {
                                 title: 'MLflow',
                                 link: '/en-us/docs/dev/user_doc/guide/task/mlflow.html',
                             },
+                            {
+                                title: 'Openmldb',
+                                link: '/en-us/docs/dev/user_doc/guide/task/openmldb.html',
+                            },
```

Comment:
You have to add zh docs into https://github.com/apache/dolphinscheduler/blob/dev/docs/configs/docsdev.js#L528 you could search `MLflow` in file https://github.com/apache/dolphinscheduler/blob/dev/docs/configs/docsdev.js and there are four match item here